lib/rubocop/rspec/shared_contexts.rb

```diff
@@ -136,11 +136,13 @@ def source_range(range, buffer: source_buffer)
       rails_version || RuboCop::Config::DEFAULT_RAILS_VERSION
     )
 
-    allow(config).to receive(:gem_versions_in_target).and_return(
-      {
-        'railties' => rails_version_in_gemfile
-      }
-    )
+    allow(config).to receive(:gem_versions_in_target).and_wrap_original do |method|
```

Comment:
This was too specific and would discard gems from a provided lockfile in tests. Instead, modify the original. This change works fine for `rubocop-rails` specs.